health-services/household/src/main/resources/household-persister.yml

```
@@ -163,4 +196,12 @@
             - jsonPath: $.*.rowVersion
             - jsonPath: $.*.isDeleted
-            - jsonPath: $.*.id
+            - jsonPath: $.*.id
+        - query: UPDATE HOUSEHOLD_MEMBER_RELATIONSHIP SET lastModifiedBy = ?, lastModifiedTime = ?, rowVersion = ?, isDeleted = ? WHERE householdMemberId = ?;
+          basePath: $.*.relationships.*
+          jsonMaps:
+            - jsonPath: $.*.relationships.*.auditDetails.lastModifiedBy
+            - jsonPath: $.*.relationships.*.auditDetails.lastModifiedTime
+            - jsonPath: $.*.relationships.*.rowVersion
+            - jsonPath: $.*.relationships.*.isDeleted
+            - jsonPath: $.*.relationships.*.householdMemberId
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)